probe-rs/targets/GD32F10x_Series.yaml

```diff
@@ -0,0 +1,3354 @@
+name: GD32F10x Series
+generated_from_pack: true
+pack_file_release: 2.0.3
+variants:
+- name: GD32F101C4
+  cores:
+  - name: main
+    type: armv7m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Ram
+    name: IRAM1
+    range:
+      start: 0x20000000
+      end: 0x20001000
+    cores:
+    - main
+    access:
```

Comment:
RAM not having `execute: true` means probe-rs won't be able to flash these devices.